conf/app.routes

```diff
@@ -111,3 +111,8 @@ POST        /remove-uk-trading-name/:index                       controllers.tra
 
 GET         /remove-all-trading-names                            controllers.tradingNames.DeleteAllTradingNamesController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints)
 POST        /remove-all-trading-names                            controllers.tradingNames.DeleteAllTradingNamesController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints)
+
+GET         /bank-details                                        controllers.BankDetailsController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints)
+POST        /bank-details                                        controllers.BankDetailsController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints)
+GET         /check-bank-details                                  controllers.BankDetailsController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints)
+POST        /check-bank-details                                  controllers.BankDetailsController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints)
```

Comment:
Do we need the `/check-bank-details` routes ?

Reply:
No they can be removed.